dev-libs/userspace-rcu/files/userspace-rcu-0.13.1-loong.patch

```diff
@@ -0,0 +1,177 @@
+This is https://github.com/loongarch64/userspace-rcu/pull/2, yet to be
+upstreamed but already reviewed and accepted by the LoongArch porting
+community.
+From: Wang Jing <wangjing@loongson.cn>
```

Comment:
For unmerged PRs, I keep the commit hash so we know which version, but I guess the date is good enough